tools/test/h5repack/h5repack.sh.in

```diff
@@ -1681,6 +1709,10 @@ TOOLTEST HDFFV-5932 h5repack_attr_refs.h5
 # Note: this test is experimental for sharing test file among tools
 TOOLTEST HDFFV-7840 h5diff_attr1.h5
 
+# test HDFFV-10590
+arg="HDFFV-10590_CVE-2018-17432.h5 HDFFV-10590_CVE-2018-17432_out.h5 --low=1 --high=2 -f GZIP=8 -l dset1:CHUNK=5x6"
+TOOLTEST_FAIL $arg
+
```

Comment:
I don't have a strong feeling, but why the different argument handling for this test?

Reply:
The test was provided by the user.